src/app/public/modules/modal/modal.interface.ts

```diff
@@ -1,10 +1,69 @@
+// TODO: defaults won't show in the generated docs until this work is done:
+// https://github.com/blackbaud/skyux-docs-tools/issues/38
+
+/**
+ * Specifies configuration options for creating a modal.
+ */
 export interface SkyModalConfigurationInterface {
+
+  /**
+   * Indicates whether to display the modal full screen.
+   * This property defaults to `false`.
+   */
   fullPage?: boolean;
+
+  /**
+   * Specifies a size for the modal. The valid options are `small`, `medium`, and `large`.
+   * If no value is provided, the size will be set to `medium`.
+   */
   size?: string;
+
+  /**
+   * An array property of `providers`.
```

Comment:
In the current docs, we indicate that `providers` is required, but we don't here. I assume the current docs are wrong based on the `?` with the property in the code, but I want to make sure.

Reply:
That's correct. I also thought it was strange and double checked it - you can still open modals without using this property. It is optional.